tools/ci-build/changelogger/src/new.rs

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+use anyhow::Context;
+use clap::Parser;
+use smithy_rs_tool_common::changelog::{FrontMatter, Markdown, Reference, Target};
+use smithy_rs_tool_common::git::find_git_repository_root;
+use smithy_rs_tool_common::here;
+use std::path::PathBuf;
+
+#[derive(Parser, Debug, Eq, PartialEq)]
+pub struct NewArgs {
+    /// Target audience for the change (if not provided, user's editor will open for authoring one)
+    #[clap(long, short = 't')]
+    pub applies_to: Option<Vec<Target>>,
+    /// List of git usernames for the authors of the change (if not provided, user's editor will open for authoring one)
+    #[clap(long, short)]
+    pub authors: Option<Vec<String>>,
```

Comment:
Nit: It would be nice if I didn’t have to specify this manually and if the tool could automatically determine it from, for example, `git config user.email`. We would likely need to update the `smithy-rs-maintainers.txt` file to include a format that represents email addresses and aliases for each user.

Reply:
Yeah, that'd be nice. Sometimes there are multiple authors for an entry, though, so I'd defer implementing this nice-to-have.